[MM-1435]: Added the test cases for getting notification for draft PRs

arush-vashishtha opened the pull request:
#### Summary
This PR consists the test cases for the following scenarios,

1. Notifications on MM for draft and normal Pull Requests from GitHub.
2. Notifications when a draft Pull Request is marked as ready for review.

cwarnermm committed:
Replaced legacy GitBook URLs with PL links (#126)

* Replaced legacy GitBook URLs with PL links

* Added PL link for Confluence

yasserfaraazkhan committed:
chore: data sync

yasserfaraazkhan committed:
sync: data

yasserfaraazkhan committed:
Feat(IP Filtering): Add test cases (#133)

yasserfaraazkhan committed:
feat: mobile tests for GM to channel (#128)

lindalumitchell committed:
test(MM-T1254): Updating behavior of CTRL/CMD-down in text input per MM-55653 (#137)

* Update MM-T1257.md

Regression behavior is actually more consistent and desirable, so we're updating test cases to reflect that CTRL/CMD+down *should* clear the text box when there aren't any more posts to cycle through.

* Update MM-T1254.md

Updating RF tests and our test cases to reflect newly desired behavior in https://mattermost.atlassian.net/browse/MM-55653

* Update MM-T1256.md

Updating based on discussion around https://mattermost.atlassian.net/browse/MM-55653. This case though doesn't clear the text field; it moves the cursor. Updated to clarify.

---------

Co-authored-by: yasserfaraazkhan <attitude3cena.yf@gmail.com>

saturninoabril committed:
chore(data): sync

AayushChaudhary0001 committed:
[MI-3752-1]: Added the test cases for text formatting on MS Teams sync plugin. (#14) (#136)

* [MI-3752-1]: Added the test cases for text formatting on MS Teams sync plugin

* [MI-3752-1]: Fixed multiple comments for the test cases steps and values.

* [MI-3752-1]: Fixed file name.

* [MI-3752-1]: refactored the test steps as discussed.

* [MI-3752-1]: Fixes for the test cases.

Co-authored-by: Arush Vashishtha <145128655+arush-vashishtha@users.noreply.github.com>

AayushChaudhary0001 committed:
[MI-3752]: Added the test cases for the mentions in MS Teams sync plugin (#13) (#135)

* [MI-3752]: Added the test cases for the mentions in MS Teams sync plugin

* [MI-3752]: Fixed multiple comments regarding the steps and values.

* [MI-3752]: Added the special mentions for channel and here.

* [MI-3752]: Fixed the comments for merging the steps of mention and send.

* [MI-3752]: Updated the test cases according to the comments.

* [MI-3752]: changed the special mentions.

* [MI-3752]: changed the single user test case.

* [MI-3752]: Added the fixes for the test cases.

Co-authored-by: Arush Vashishtha <145128655+arush-vashishtha@users.noreply.github.com>

AayushChaudhary0001 committed:
[MI-3708]: Added the test cases for attachment sync for MS Teams sync plugin. (#12) (#134)

* [MI-3626]: Test cases of sloash commands for MS Teams plugin

* Review fixes



* [MI-3626]: Review fixes of this PR

* [MI-3708]: Added the test cases for attachment sync on MS Teams sync plugin.

* [MI-3708]: Fixed multiple comments for steps and values.

* Update Not_allowed_attachment_size.md

* Update Not_allowed_attachment_size.md

* [MI-3708]: Fixed the comments.

* [MI-3708]: solved the conflicts.

* [MI-3708]: Combined the steps for Upload and send attachment.

* [MI-3708]: Fixed the comments and added the test case for Multiple attachments with not allowed size attachment.

* [MI-3708]: added the fixes for the test cases.

* Fixed the review fixes



---------

Co-authored-by: Arush Vashishtha <145128655+arush-vashishtha@users.noreply.github.com>
Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

AayushChaudhary0001 committed:
Added the test cases for slash commands for MS Teams sync plugin (#131)

* [MI-3655]: Added test cases for message edit feature in MS Teams (#8)

* [MI-3655]: Added test cases for message edit feature in MS Teams

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Updated the mention edit test case according to the review fix

Co-authored-by: Manoj Malik <manoj@brightscout.com>

---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>
Co-authored-by: Manoj Malik <manoj@brightscout.com>

* [MI-3653]: Test cases of message sync of MS Teams (#7)

* [MI-3653]: Test cases of message sync in MS Teams

* [MI-3653]: Added the test cases for message sync

* [MI-3653]: Fixed the review comments

* [MI-3653]: Removed the extra files of slash commands

* [MI-3653]: Fixed the folders in the test cases

* [MI-3653]: Added the test case of order of posts

* [MI-3653]: Added the correct name of the plugin as review fixes

* [MI-3653]: Fixed the folder name of the plugin

* [MI-3626]: Test cases of slash commands for MS Teams plugin (#6)

* [MI-3626]: Test cases of sloash commands for MS Teams plugin

* Review fixes

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* [MI-3626]: Review fixes of this PR

* [MI-3626]: Fixed the review comments of slash commands and permissions

* [MI-3626]: Fixed the review fixes of plugin name

* [MI-3626]: Fixed the folder names

---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Fixed the folder name of message edit feature (#10)

* Fixed the folder name

* Fixed the review fixes in this PR

* Review fixes

* Review fixes new

* Changed the team name in all test cases (#11)

* Added the test cases for slash commands for MS Teams Sync plugin

---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>
Co-authored-by: Manoj Malik <manoj@brightscout.com>

AayushChaudhary0001 committed:
Added the test cases of message sync for MS Teams Sync plugin (#130)

* [MI-3655]: Added test cases for message edit feature in MS Teams (#8)

* [MI-3655]: Added test cases for message edit feature in MS Teams

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Updated the mention edit test case according to the review fix

Co-authored-by: Manoj Malik <manoj@brightscout.com>

---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>
Co-authored-by: Manoj Malik <manoj@brightscout.com>

* [MI-3653]: Test cases of message sync of MS Teams (#7)

* [MI-3653]: Test cases of message sync in MS Teams

* [MI-3653]: Added the test cases for message sync

* [MI-3653]: Fixed the review comments

* [MI-3653]: Removed the extra files of slash commands

* [MI-3653]: Fixed the folders in the test cases

* [MI-3653]: Added the test case of order of posts

* [MI-3653]: Added the correct name of the plugin as review fixes

* [MI-3653]: Fixed the folder name of the plugin

* [MI-3626]: Test cases of slash commands for MS Teams plugin (#6)

* [MI-3626]: Test cases of sloash commands for MS Teams plugin

* Review fixes

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* [MI-3626]: Review fixes of this PR

* [MI-3626]: Fixed the review comments of slash commands and permissions

* [MI-3626]: Fixed the review fixes of plugin name

* [MI-3626]: Fixed the folder names

---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Fixed the folder name of message edit feature (#10)

* Fixed the folder name

* Fixed the review fixes in this PR

* Review fixes

* Review fixes new

* Changed the team name in all test cases (#11)

* Added the test cases for message sync for MS Teams sync plugin

---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>
Co-authored-by: Manoj Malik <manoj@brightscout.com>

AayushChaudhary0001 committed:
Added the test cases for message edit for MS Teams Sync plugin (#129)

* [MI-3655]: Added test cases for message edit feature in MS Teams (#8)

* [MI-3655]: Added test cases for message edit feature in MS Teams

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Review comments

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Updated the mention edit test case according to the review fix

Co-authored-by: Manoj Malik <manoj@brightscout.com>

---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>
Co-authored-by: Manoj Malik <manoj@brightscout.com>

* [MI-3653]: Test cases of message sync of MS Teams (#7)

* [MI-3653]: Test cases of message sync in MS Teams

* [MI-3653]: Added the test cases for message sync

* [MI-3653]: Fixed the review comments

* [MI-3653]: Removed the extra files of slash commands

* [MI-3653]: Fixed the folders in the test cases

* [MI-3653]: Added the test case of order of posts

* [MI-3653]: Added the correct name of the plugin as review fixes

* [MI-3653]: Fixed the folder name of the plugin

* [MI-3626]: Test cases of slash commands for MS Teams plugin (#6)

* [MI-3626]: Test cases of sloash commands for MS Teams plugin

* Review fixes

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* [MI-3626]: Review fixes of this PR

* [MI-3626]: Fixed the review comments of slash commands and permissions

* [MI-3626]: Fixed the review fixes of plugin name

* [MI-3626]: Fixed the folder names

---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

* Fixed the folder name of message edit feature (#10)

* Fixed the folder name

* Fixed the review fixes in this PR

* Review fixes

* Review fixes new

* Changed the team name in all test cases (#11)

* Added the test cases for message edit in MS Teams Sync plugin

---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>
Co-authored-by: Manoj Malik <manoj@brightscout.com>

arush-vashishtha committed:
[MI-3769]: Added the test cases for replies on MS Teams sync plugin. (#15) (#138)

* [MI-3769]: Added the test cases for replies on MS Teams sync plugin.

* [MI-3769]: refactored the test cases.

* [MI-3769]: Fixed comments on the pr.

* [MI-3769]: fixed the comments on the test cases for replies.

yasserfaraazkhan committed:
chore: Data sync

lindalumitchell committed:
test(feature): Added test for LaTeX currently not expected to render in MS Teams (#139)

* Added test for LaTeX formatting in MS Teams

* Remove team not in expected team names

* Add second scenario

* Update data/test-cases/plugins/ms-teams-sync/text-formatting/Formatting_latex.md

Co-authored-by: Daniel Espino García <larkox@gmail.com>

---------

Co-authored-by: Daniel Espino García <larkox@gmail.com>

yasserfaraazkhan committed:
chore: Data sync

yasserfaraazkhan committed:
Feat: wrangler test cases (#142)

yasserfaraazkhan committed:
chore: Data sync

saturninoabril committed:
chore(data): sync

DHaussermann committed:
Show that LDAP users cannot be deactivated (#124)

* Show that LDAP users cannot be deactivated

* Fixed the typos in test scripr.

saturninoabril committed:
chore(data): sync

varghesejose2020 committed:
Tested Remove button in System console T955 (#123)

* Tested Custom Emoji T2184

Tested Custom Emoji in Channel for the PR
[MM-PR-24785](https://github.com/mattermost/mattermost/pull/24785)

* Update MM-T2184.md

* Update MM-T2184.md

* Tested system console  remove buttonT955

Tested  remove button component in system console for the PR
[MM-PR-24912](https://github.com/mattermost/mattermost/pull/24912#issuecomment-1773714773)

---------

Co-authored-by: Saturnino Abril <saturnino.abril@gmail.com>

saturninoabril committed:
chore(data): sync

mkaraki committed:
Add test description for WEBP upload test (#150)

saturninoabril committed:
chore(data): sync

yasserfaraazkhan committed:
[MM-56320] Create test cases for user limit warnings (#147)

yasserfaraazkhan committed:
User Management Test cases (#162)

yasserfaraazkhan committed:
chore(sync): data

yasserfaraazkhan committed:
Add test case for non channel admin accessing post links (#163)

* add test case for non channel admin accessing post links

yasserfaraazkhan committed:
Create test cases for auto-mute/unmute channels (#151)

yasserfaraazkhan committed:
chore(sync): data

arush-vashishtha committed:
[MI-3775]: Added test cases for reactions on MS Teams sync plugin. (#16) (#140)

* [MI-3775]: Added test cases for reactions on MS Teams sync plugin.

* [MI-3775]: changed the name in the file for 2 test cases.

* [MI-3775]: Refactored test cases.

* [MI-3775]:Fixed the comments on the test cases and changed the folder name.

* [MI-3775]: changes in the name.

* [MI-3775]: Fixed the comments by Ayush.

* [MI-3775]: fixed the comments.

* [MI-3775]: Fixed the comments.

Co-authored-by: Raghav Aggarwal <raghav.aggarwal@brightscout.com>

arush-vashishtha committed:
[MI-3783] : Added the test cases for the Plugin Configurations on MM. (#22) (#141)

* [MI-3783]: Added the test cases for the plugin configuration on MS Teams Sync plugin.

* [MI-3783]: fixed multiple comments by Aayush.

* [MI-3783]: Fixed the comments.

* [MI-3783]: fixed the comments.

* [MI-3783]: fixed the the indexing.

* [MI-3782]: Added the test cases for auto promote user and fixed the comments.

* [MI-3783]: added the scenario.

* [MI-3783]: fixed the comments on the test cases.

* Update data/test-cases/plugins/ms-teams-sync/plugin-configurations/Certificate_based_subscriptions.md



---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

arush-vashishtha committed:
[MI-3804]: Added the test cases for the UI design on MS Teams sync plugin. (#19) (#143)

* [MI-3804]: Added the test cases for the UI design on MS Teams sync plugin.

* [MI-3804]: added the test case for disconnected user with linked channels.

* [MI-3804]:Added the changes.

* [MI-3804]: added the changes for disconnecting user.

* [MI-3804]: Fixed the commetns by Aayush on the test cases.

* [MI-3804]: Fixed the comments.

* [MI-3804]: fixed the comments by Ayush.

* changed the test case.

arush-vashishtha committed:
[MI-3812]: Added the test cases for UI of list of linked channels in RHS on MM. (#20) (#144)

* [MI-3812]: Added the test cases for list of linked channels UI on MS Teams sync plugin.

* [MI-3812]: added few updates in the PR.

* changed the folder name

* Fixed the comments.

* changes the folder name.

* fixed the minor comments by Aayush.

* fixed the folder name and added the test scenario.

* fixed

* fixed

* fixed the comments.

* fixed the comments.

* fixed the values and comments.

* added the scenario.

* Update data/test-cases/plugins/ms-teams-sync/ui-integration/List_linked_channels.md



---------

Co-authored-by: Ayush Thakur <100013900+ayusht2810@users.noreply.github.com>

arush-vashishtha committed:
[MI-3823]: Added the test cases for linking and unlinking channel on MM. (#23) (#145)

* [MI-3823]: Added the test cases for phase 3.

* [MI-3823]: Added the test cases for linking and unlinking channels on MM.

* [MI-3823]: refactored the scenarios.

* .

* changed the folder name

* changed the folder name.

* fixed the comments.

* changed the folder name.

* fixed the comments.

* fixed the unlink feature and added a new scenario.

* fixed the comments.

* fixed the comments.

lindalumitchell committed:
Update MM-T847.md (#164)

Remove extra info about channel order, which isn't the core of this test case and depends on config anyway.

yasserfaraazkhan committed:
commit after deno validate-all

yasserfaraazkhan committed:
Sync: data

yasserfaraazkhan committed:
Test cases for User limit enforcement (#167)

lindalumitchell committed:
Update teamOptions list (#169)

* Update team list

Updating teamOptions list to add current teams available in Jira. Did not remove old teams as that could affect existing tests; those should be updated in a targeted way.

Adding:
```
ABC
Cloud Platform
Customer Reliability Engineering
Deployment Engineering
Integrations
XYZ
```

* Add further team names

saturninoabril committed:
chore: data sync

saturninoabril committed:
chore: remove codeowner except for .github/workflows (#170)

lieut-data committed:
test(MS Teams plugin): bot messages for v1.10 (#168)

* Document bot messages from MS Teams v1.10

* s/Very/Verify/g

* use Preconditions vs multiple Steps

---------

Co-authored-by: Saturnino Abril <saturnino.abril@gmail.com>

saturninoabril committed:
chore(data):sync

yasserfaraazkhan committed:
Sync: data

saturninoabril committed:
chore(data): sync

yasserfaraazkhan committed:
Data(sync)

arush-vashishtha committed:
[MM-33]: Added the test cases for the tooltip upon hovering the cursor over the issue link in the channel or DM/GM on MM. (#24) (#148)

* [MM-33]: Added the test cases for the tooltip upon hoveing the cursor over the issue link in the channel or DM/GM on MM.

* [MM-33]: edited the folder name.

* [MM-33]: fixed the comments by Aayush.

saturninoabril committed:
chore(data): sync

yasserfaraazkhan committed:
chore(data): Sync

yasserfaraazkhan committed:
User Survey plugin test cases (#166)

saturninoabril committed:
chore(data): sync (#171)

phoinixgrr committed:
[chore] migration of plugin-store (#172)

Plugin store is gradually migrated from:
- https://plugins-store.test.mattermost.com
to
- https://plugins.releases.mattermost.com

We reflect that change here

Note: Currently both CDN's are working as expected, to facilitate the mgiration. Upon succesfull migration, https://plugins-store.test.mattermost.com will be decomissioned

Signed-off-by: Akis Maziotis <akis.maziotis@mattermost.com>

saturninoabril committed:
chore(data): sync (#173)

yasserfaraazkhan committed:
Add test cases for Admin control for User settings (#174)

saturninoabril committed:
chore: data sync (#175)

saturninoabril committed:
chore: data sync (#176)

saturninoabril committed:
feat(test): attachments removal plugin (#177)

* feat(test): attachments removal plugin

* update hosting to self-hosted only

saturninoabril committed:
chore: data sync (#178)

saturninoabril committed:
feat(attachment remover): hide option if no permission nor outside edit time limit (#179)

saturninoabril committed:
chore: data sync (#180)

DHaussermann committed:
Add tests for Global Retention policy in hours (#181)

* Test cases for Global Retetion in hours

* Test cases for Global Retetion in hours

* Correct typos

* As per Deno - formating for italic text requires correction

* Make 2 of the steps more explicit to avoid possibility overlapping scenarios hidding a bug.

* Update data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForAttachmentInHours.md

Co-authored-by: Saturnino Abril <5334504+saturninoabril@users.noreply.github.com>

* Update data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForAttachmentInHours.md

Co-authored-by: Saturnino Abril <5334504+saturninoabril@users.noreply.github.com>

* Update data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForMessagesInHours.md

Co-authored-by: Saturnino Abril <5334504+saturninoabril@users.noreply.github.com>

* Update data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForMessagesInHours.md

Co-authored-by: Saturnino Abril <5334504+saturninoabril@users.noreply.github.com>

---------

Co-authored-by: Saturnino Abril <5334504+saturninoabril@users.noreply.github.com>

saturninoabril committed:
chore: data sync (#185)

* chore: data sync

* more sync

* more sync

yasserfaraazkhan committed:
Feat: Test cases for Notification test tool (#187)

yasserfaraazkhan committed:
Feat: Mobile Drafts (#186)

yasserfaraazkhan committed:
Feat: Make desktop app landing page optional (#189)

saturninoabril committed:
chore: data sync (#190)

saturninoabril committed:
upgrade Deno to 2.0 (#191)

yasserfaraazkhan committed:
Feat: Notifications Banner (#184)

yasserfaraazkhan committed:
feat: Test cases for Scheduled Message feature (#183)

yasserfaraazkhan committed:
Feat: Add test case for Copying contents of scheduled message (#193)

* Feat: Add test case for Copying contents of scheduled message

yasserfaraazkhan committed:
add missing steps

saturninoabril committed:
chore: data sync (#194)

yasserfaraazkhan committed:
Feat(Edit Attachments): Add tests cases for edit attachments (#196)

saturninoabril committed:
data sync and fix frontmatter formatting (#197)

yasserfaraazkhan committed:
Sync: data

yasserfaraazkhan committed:
Sync: Data

- Add back the steps in data/test-cases/channels/edited-post that was skipped due to removing **Step 1** from file.

Victor-Nyagudi committed:
Add test case for DM list scrolling behavior (#198)

Co-authored-by: Mattermost Build <build@mattermost.com>

saturninoabril committed:
chore(data): sync

yasserfaraazkhan committed:
Updates zephyr test cases for edit file attachment (#201)

yasserfaraazkhan committed:
Test case for correct mentions count when user follows/unfollows thread (#202)

* Test case for correct mentions count when user follows/unfollows thread

yasserfaraazkhan committed:
feat(Scheduled messages): Mobile scheduled messages (#200)

yasserfaraazkhan committed:
Sync: data (#203)

saturninoabril committed:
chore: data sync (#205)

* chore: data sync

* Update MM-T5752.md

Added GitHub username in `authors` field and Workflows team in `team_ownership` field

* Update MM-T5751.md

Added GitHub username in `authors` field and Workflows team in `team_ownership` field

* Update MM-T5750.md

Added GitHub username in `authors` field and Workflows team in `team_ownership` field

* Update MM-T5749.md

Added GitHub username in `authors` field and Workflows team in `team_ownership` field

* Update MM-T5748.md

Added GitHub username in `authors` field and Workflows team in `team_ownership` field

* Update MM-T5747.md

Added GitHub username in `authors` field and Workflows team in `team_ownership` field

* Update MM-T5746.md

Added GitHub username in `authors` field and Workflows team in `team_ownership` field

* Update MM-T5745.md

Added GitHub username in `authors` field and Workflows team in `team_ownership` field

* fix validation

---------

Co-authored-by: lindy65 <lindy65@users.noreply.github.com>

saturninoabril committed:
MM-63581 Remove images from a bucket (#206)

* chore: data sync

* remove images using s3 bucket

saturninoabril committed:
feat(custom profile attributes): user settings (#210)

saturninoabril committed:
chore: data sync (#211)

* chore: data sync

* Update data/test-cases/suite/system-console/site-configuration/system-properties--custom-profile-attributes-/system-console/MM-T5766.md

Co-authored-by: yasser khan <attitude3cena.yf@gmail.com>

---------

Co-authored-by: yasser khan <attitude3cena.yf@gmail.com>

saturninoabril committed:
feat(custom profile attributes): profile popover (#212)

saturninoabril committed:
chore (custom profile attributes): data sync (#213)

* chore: data sync

* add metafiles

yasserfaraazkhan committed:
Mobile edit files (#217)

yasserfaraazkhan committed:
Mobile Playbook Test cases (#207)

yasserfaraazkhan committed:
Sync(Data): Sync test cases

lindalumitchell committed:
Feat(ABAC-channel-admin): Adding ABAC channel admin test cases (#218)

lindalumitchell committed:
test(ABAC-system-admin): Add ABAC system admin test cases (#219)

yasserfaraazkhan committed:
Feat: Test cases for Mobile Permalink preview (#220)

* Feat: Test cases for Mobile Permalink preview

yasserfaraazkhan committed:
Run Save And Sync after merging to Main (#223)

yasserfaraazkhan committed:
Chore: Sync Data

arush-vashishtha committed:
[MM-1435]: added the test cases for getting notification for draft PRs

arush-vashishtha committed:
reprashed the test cases

arush-vashishtha committed:
reprashed and added a testcase

arush-vashishtha committed:
Fixed the spelling error

Yash-Pathak-03 commented on data/test-cases/plugins/github/notifications/Notification_for_Draft_PR.md:
```diff
@@ -0,0 +1,95 @@
+
+**Expected**
+
+The user should get a complete notification in Mattermost with all details when the draft `Pull Request` is marked as ready for review.
```
Mention the expected fields to be shown in notification for normal, draft and marked ready for review PRs.

Yash-Pathak-03 commented on data/test-cases/plugins/github/notifications/Notification_for_Draft_PR.md:
```diff
@@ -0,0 +1,95 @@
+
+**Step 1**
+
+1. Connect your Mattermost account to your `GitHub` account and create a `subscription` for `Pull Request` in any channel.
```
Add a testcase when draft PR is marked closed and then reopened with the config setting enabled/disabled

Yash-Pathak-03 commented on data/test-cases/plugins/github/notifications/Notification_for_Draft_PR.md:
```diff
@@ -0,0 +1,95 @@
+---
+# (Required) Ensure all values are filled up
+name: "Getting notifications on MM for draft `Pull Request`s from GitHub"
```
Don't use abbreviations in title write complete mattermost instead of MM, capitalise the first letter in the PR heading and correct the capital letters used in the middle of sentences.

Yash-Pathak-03 reviewed (changes requested):
Please look into this

saturninoabril committed:
chore: sync data

arush-vashishtha committed:
Added the test cases for the mentions and labels/ milestones displayed in RHS of github (#231)

* [MM-130]: Added the test cases for the mentions and labels/ milestones displayed in RHS. (#40)

Summary
This PR consists the test cases for the following scenarios,

DM notification for mentions/assigns.
Labels/Milestones added to the PR/issues on Github.

* Review fixes

saturninoabril committed:
chore: data sync

arush-vashishtha committed:
Review fixes

Commit:
Merge branch 'testcases' into MM-1435
